code/main/edge/stateStoreManager.brs

```diff
@@ -64,67 +70,67 @@ function _adb_StateStoreManager() as object
                 return
             end if
 
-            m._stateStoreMap[payload.key] = _adb_StateStoreEntry(payload)
+            maxAgeSeconds = payload.maxAge
+            if not _adb_isPositiveWholeNumber(maxAgeSeconds)
+                _adb_logDebug("_adb_StateStore::_setExpiryTime() - Invalid payload.maxAge value:(" + FormatJson(maxAgeSeconds) + "), using 0 as default value.")
+                maxAgeSeconds = 0
```

Comment:
Is there a default max age? You can remove the entry instead of setting its age to 0, as it will simply expire during the next call.

Reply:
@praveek Updated the logic to explicitly delete the entry.